content-src/components/Card/Card.jsx

```diff
@@ -245,10 +246,8 @@ export class _Card extends React.PureComponent {
           </div>
         </div>
       </a>
-      {!props.placeholder && <button className="context-menu-button icon" title={this.props.intl.formatMessage({id: "context_menu_title"})}
-        onClick={this.onMenuButtonClick}>
-        <span className="sr-only">{`Open context menu for ${link.title}`}</span>
-      </button>}
+      {!props.placeholder && <button aria-haspopup="true" aria-label={this.props.intl.formatMessage({id: "context_menu_button_sr"}, {title})} className="context-menu-button icon" title={this.props.intl.formatMessage({id: "context_menu_title"}, {title})}
```

Comment:
@AllegroFox In title={this.props.intl.formatMessage({id: "context_menu_title"}, {title})} , we can omit title and should be title={this.props.intl.formatMessage({id: "context_menu_title"})}